### ISSUE TITLE
ddox problems

### ISSUE THREAD

**Issue by John-Colvin** [DlangScience/DlangScience.github.io#10]
E.g. https://dlangscience.github.io/dstats/api/dstats/summary/geometricMean.html is 404 and is linked from https://dlangscience.github.io/dstats/api/dstats/summary.html

**Comment by John-Colvin** [comment]
https://github.com/rejectedsoftware/ddox/issues/91

**Comment by 9il** [comment]
I like https://github.com/kiith-sa/harbored-mod

**Comment by ghost** [comment]
I don't know enough about ddox vs. harbored-mod to give feedback, but I think this should be discussed/decided right away, no?

**Comment by 9il** [comment]
No, we can migrate later.

**Comment by John-Colvin** [comment]
Yes, we should stick with ddox for now. Swapping from harbored-mod back to ddox would be hard if we e.g. used ant of its markdown syntax, but ddox -> harbored-mod should be easy if we find we need to at a later date.

**Comment by ghost** [comment]
Okay. I was exactly thinking of markdown within ddoc as a unified solution for general site and api docs, but maybe it's too soon to head down that road.

**Comment by ghost** [comment]
Concerning rejectedsoftware/ddox#91, does this [post](http://forum.dlang.org/post/tcunkoqxnzcsjzfajjvn@forum.dlang.org) show a general way to specify ddox options when using dub build?

**Comment by John-Colvin** [comment]
@jansengb only for the filter stage. dub is rubbish for configuring ddox, I've rolled our own solution (see e.g. https://github.com/DlangScience/clFFT-D/pull/4)
